draftjs_exporter/style_state.py

```diff
@@ -57,12 +58,19 @@ def get_style_value(self):
         return ''.join(sorted(rules))
 
     def add_node(self, element, text):
+
+        for deco in self.composite_decorators:
+            text = deco.process(text, parent=element)
+
+        text_children = list(DOM.parse_html(
+            '<textnode>' + text + '</textnode>').body.children)
```

Comment:
You are right, if we have to do this, it's better to move the internal detail in `DOM`. But maybe it's better not to use string parsing at all. I'll try that.